modules/ROOT/nav.adoc

```diff
@@ -1,41 +1,41 @@
 .{product}
-* xref:introduction.adoc[]
-* xref:components.adoc[]
+* xref:ROOT:introduction.adoc[]
```

Comment:
I wasn't able to leave a comment on the line for sideloader so I'm leaving it here. Why isn't sideloader first in the nav? It's the new and cool tool. I noticed that the hero section is for ZDM, too. Was that a PM's choice?

Reply:
Sideloader is being presented as part of ZDM. This is the marketing page for it: https://www.datastax.com/products/datastax-astra/migration-toolkit

Reply:
That's not how it's presented in the docs, though.

Reply:
Please see updated preview. The narrative of the 4 tools should be stronger now. Sideloader is higher up in the nav.

Reply:
@KimberlyFields  Please see the following:

* Astra DB Sideloader is first after ZDM proxy on the index page https://d5rxiv0do0q3v.cloudfront.net/doc-5183/data-migration/index.html
* New Get started section that compares the tools and establishes the relationship between ZDM and Sideloader. https://d5rxiv0do0q3v.cloudfront.net/doc-5183/data-migration/components.html